tests/vmi_multus_test.go

```diff
@@ -941,6 +942,14 @@ var _ = Describe("[Serial]Macvtap", func() {
 		return vmi
 	}
 
+	createCirrosVMIWithMacvtapDefinedMAC := func(virtClient kubecli.KubevirtClient, networkName string, mac string) *v1.VirtualMachineInstance {
```

Comment:
Why is this new method required? You can set a mac on `createCirrosVMIWithMacvtapStaticIP`.

Reply:
Yes I can, but I don't care at all about setting an IP on it.

Reply:
You can add a new method `createCirrosVMIWithMacvtapDefinedMAC` but at least reuse the extended existing one (`createCirrosVMIWithMacvtapStaticIP`).

Reply:
Nope, I can't.

As per https://github.com/kubevirt/kubevirt/pull/2935#discussion_r505493506, a VM pinned to a node is not able to migrate. 

As such, I'd need a new method anyway. Might as well let it do what I need it to do: just set a MAC address.

Reply:
I meant that if the `nodeName` is empty the general method won't pin the vm to the node.
But up to you, it you think two method is more readable, go for it.

Reply:
It doesn't work without updating the implementation of the `StartVmOnNode` [function](https://github.com/kubevirt/kubevirt/blob/4791356299f1ae452b2873e2f1f236d00233ded9/tests/utils.go#L3800). As you can see, the NodeAffinity is set independently of the nodeName variable.

I'd rather add a new helper than making the  `StartVmOnNode` work with an empty node.